scripts/docker/cpp-build-env.dockerfile

```diff
@@ -0,0 +1,30 @@
+# build:          docker build -t test_build_20 .
+# run:            docker run -it --rm test_build_20:latest
+# versioning:     docker tag cpp-build-env ethereum/cpp-build-env:v0.8.29.2017
+# upload:         docker push tcsiwula/cpp-build-env
+
+# get latest stable debian release image -- see https://hub.docker.com/_/debian/
+from debian:latest
+
+# maintainer email
+MAINTAINER Tim Siwula <tcsiwula@gmail.com>
+
+# by defualt, docker runs all processes as root within the container
+USER root
```

Comment:
This is not needed unless we want to change the user. I think we should.

Reply:
For security reasons I assume, I changed it. I think most machines or the ones that I have used require  sudo for apt-install commands...

Reply:
What do you think about this, for best for this? @chfast https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo